src/mpid/ch4/src/ch4_coll_globals_default.c.in

```diff
@@ -51,6 +57,13 @@ const MPIDI_coll_algo_container_t CH4_Bcast_intra_composition_gamma_cnt = {
                                                 }
 };
 
+const MPIDI_coll_algo_container_t CH4_Bcast_intra_composition_delta_cnt = {
+    .id = MPIDI_CH4_Bcast_intra_composition_delta_id,
+    .params.ch4_bcast_params.ch4_bcast_delta = {
+                                                .bcast = -1,
+                                                }
```

Comment:
Hmmm. Looks like if you had one extra line, then you would make code_cleanup more adequate here. I would remove next like here or added next line in all other places :)